### DISCUSSION TITLE
Update main repository

### DISCUSSION THREAD

**Discussion by IoSonoPiero** [vladmandic/sdnext#398] [Q&A] [answered]
Hello people.
I'm an absolute noob with this argument, but I'm trying to learn as much as possible.
I installed Python, Git, and CUDA and cloned the repository to a local folder of my PC on Windows.

Now,
I can't understand in "Update" section at https://github.com/vladmandic/automatic
...
_The launcher can perform automatic update of main repository, requirements, extensions and submodules:
Main repository: Update is not performed by default, enable with --upgrade flag_
...

The question is simple: what do I have to launch adding --upgrade parameter after it to force at every start a possible upgrade?

Sorry for asking.

**Comment by SoraEnkai** [comment, marked as answer]
On windows, open the terminal (cmd/powershell etc) in `automatic` folder, target `webui.bat` and add --upgrade
Like this:
![image](https://user-images.githubusercontent.com/39128897/233829456-fd80e040-aa38-44d5-bf42-3dbad9a87062.png)

**Comment by IoSonoPiero** [comment]
Oh, thank you very much! I'll add it at each start!
